### ISSUE TITLE
More books needed

### ISSUE THREAD

**Issue by nfebe** [ch3nkula/programming-books-and-resources#7]
Books covering issues like Game Development, UML, Video Streaming, Hacking, Software Testing and more are needed. Thanks to the community!

**Comment by xSavitar** [comment]
Yes, this is also relevant. There are people who will like that area of development 👍

**Comment by ankit404** [comment]
[Algorithms and Data Structures - Alan   ##Parker.pdf](https://github.com/xSavitar/programming-books-and-resources/files/3675043/Algorithms.and.Data.Structures.-.Alan.Parker.pdf)
